apps/rewards/app/components/Panel/NewReward/NewReward.js

```diff
@@ -50,7 +49,9 @@ const messages = {
   meritTokenTransferable: () => 'Merit rewards must be non-transferable.',
   amountOverBalance: () => 'Amount must be below the available balance.',
   dateStartAfterEnd: () => 'Start date must take place before the end date.',
-  singleDisbursement: () => 'While you selected a recurring dividend, based on your parameters, there will only be a single disbursement.',
+  noDisbursements: (disbursement, displayUnit) => `Based on your selected parameters, there will be no disbursements, as your end date falls before the end of the first cycle. Please choose an end date that is at least ${disbursement} ${displayUnit} after the start date.`,
+  singleDisbursement: (disbursement, displayUnit) => `There will only be a single reward under this policy, which will be disbursed ${disbursement} ${displayUnit} after your chosen start date. Dates are approximate as our disbursements occur based on block number.`,
+  multipleDisbursements: (disbursement, displayUnit) => `The first reward under this policy will be disbursed ${disbursement} ${displayUnit} after your chosen start date, and repeat every ${disbursement} ${displayUnit} until your chosen end date. Dates are approximate as our disbursements occur based on block number.`,
```

Comment:
Remove the "our"-->

Dates are approximate as ~our~ disbursements occur based on block number.